notebooks/evidence_matching/fusion_evidence_matching.ipynb

```diff
@@ -0,0 +1,396 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c75a5a00",
+   "metadata": {},
+   "source": [
+    "# Notebook for demonstrating evidence matching between assayed fusions and categorical fusions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "4ec41cf4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "from os import environ\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "environ[\"GENE_NORM_DB_URL\"] = \"postgresql://postgres@localhost:5432/gene_normalizer\"\n",
+    "environ[\"UTA_DB_URL\"] = \"postgresql://uta_admin:uta@localhost:5432/uta/uta_20240523b\""
```

Comment:
What if people have a different config but still want to run this notebook? Something we did in the manuscript repo is use python-dotenv: https://pypi.org/project/python-dotenv/

Reply:
Or just state that they should change these configs / what they are